### PR TITLE
[17.0][MIG] maintenance_timesheet: Migration to 17.0

### PR DESCRIPTION
**Pull request by FernandoRomera**

### CODE REVIEW

**Comment by dalonsod** [comment]
Hello, this PR actually depends on #401 , you can temporary add a test-requirements.txt file linking to it.

**Comment by FernandoRomera** [comment]
@dalonsod 
Thank you very much for the suggestion, but the problem lies in that  in version 17 they have removed the cron to generate maintenance requests and replaced it with activities. 
That's why tests fail in #401.
In #414 we add cron to module.

**Comment by dalonsod** [comment]
Yes, I know it and it's a todo task for #401 , not sure but it only might affect to tests. Anyway, if test-requirements.txt is added to this PR, it could help to runboat successful load and then, enable testing on it, if I'm not wrong.

**Comment by pedrobaeza** [comment]
Please check https://github.com/OCA/maintainer-tools/wiki/Use-temporary-reference(s)-to-another-pull-request(s)

**Comment by dalonsod** [comment]
@FernandoRomera could you make a rebase? `maintenance_project` is already available

**Comment by FernandoRomera** [comment]
@dalonsod 
Done.

File: maintenance_timesheet/__manifest__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "summary": "Adds timesheets to maintenance requests",
     "author": "Odoo Community Association (OCA), Solvos",
     "license": "AGPL-3",
-    "version": "16.0.1.0.1",
+    "version": "17.0.1.0.0",
     "category": "Human Resources",
     "website": "https://github.com/OCA/maintenance",
     "depends": ["base_maintenance", "maintenance_project", "hr_timesheet"],
```
Comment (dalonsod) [inline maintenance_timesheet/__manifest__.py]: Remove `base_maintenance` dependency, as already included by `maintenance_project`.

File: maintenance_timesheet/views/maintenance_request_views.xml [hunk 2]
```diff
@@ -64,7 +64,7 @@
                                 class="oe_inline"
                                 widget="timesheet_uom"
                             />
-                            <!-- t t-esc="record.timesheet_total_hours.value"/-->
+                            <!-- t t-out="record.timesheet_total_hours.value"/-->
```
Comment (dalonsod) [inline maintenance_timesheet/views/maintenance_request_views.xml]: This line might simply removed IMO

**Review by dalonsod — CHANGES_REQUESTED** [review]
Only code review, see minor changes requested. And please check tests improvement needed

**Comment by SapphireHotS** [comment]
Pressing on the "Hours" button I get Caused by: Error: "account.analytic.line"."is_timer_running" field is undefined.

**Comment by FernandoRomera** [comment]
> Pressing on the "Hours" button I get Caused by: Error: "account.analytic.line"."is_timer_running" field is undefined.

I can not reproduce it on runbot.

**Review by AntoniRomera — APPROVED** [review]
LGTM

**Review by MRomeera — APPROVED** [review]
LGTM

**Comment by SapphireHotS** [comment]
> > Pressing on the "Hours" button I get Caused by: Error: "account.analytic.line"."is_timer_running" field is undefined.
> 
> I can not reproduce it on runbot.

Maybe because I'm using enterprise?

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**